### DISCUSSION TITLE
Welcome to mystory Discussions!

### DISCUSSION THREAD

**Discussion by Sayak-Bhunia** [Sayak-Bhunia/mystory#179] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Aksshay88** [comment]
Hiya bluds 

This is Aksshay who is been an open source enthusiast and would like to contribute this open source project and I am very much interested in contributing to this Mystory .Yeah This is about me .Feel Free to raise an issue and toss any comments .
